src/migrate/Migrator.js

```diff
@@ -459,6 +528,34 @@ function validateMigrationList(migrationSource, migrations) {
   }
 }
 
+function getMigrationSubset(migrations, direction, toFile) {
+  const migrationCopy =
+    direction === 'up' ? [...migrations] : [...migrations].reverse();
+
+  if (!toFile) {
+    return migrationCopy.slice(0, 1);
+  }
+
+  const indexOfToMigration = findIndex(migrationCopy, (migration) => {
+    return migration.file === toFile;
```

Comment:
Assume we have following migrations:
- 20001101000001_initial.js
- 20001102000002_migration1.js
- 20001103000003_migration2.js

Here is we have three possible migrate 'up/down to' cases:
1. to filename - 20001102000002_migration1.js
2. to timestamp (equal to) - 20001102000002
3. to timestamp (lower / upper bound)
    - up to 20001103000000 = migrationsToRun: [20001101000001_initial.js, 20001102000002_migration1.js]
    - down to 20001102000000 = migrationsToRun: [20001103000003_migration2.js, 20001102000002_migration1.js]
    - use short form 20001103, or pattern 2000??03, or regexp: 20001103.*.js

Actually you are running migration to exactly specific file name. 
What do you think about other cases ?